src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -93,6 +100,31 @@ private Command SetupAnalyzeTemplateCommand()
             return analyzeTemplateCommand;
         }
 
+        private string GenerateResultString(Types.IEvaluation evaluation)
+        {
+            string resultString = "";
+
+            if (!evaluation.Passed)
+            {
+                if (evaluation.Results.Count() > 0)
```

Comment:
This won't display properly with evaluations that have results in addition to sub-evaluations with results.  I suggest relying on `evaluation.HasResults` when looking at each evaluation.

Reply:
Can you give an example of this? I tested it with HasResults, but for the case of allOf with two leaf expressions HasResult was true. I will just remove the check for if there is Results instead. If the count is 0, it won't loop through the for loop anyway.

Reply:
I don't understand what you mean - should allOf with two leafs *not* return true for HasResults?

Reply:
I was trying to use HasResults to know if it's a leafexpression or a structured, but it that's not what it was intended for.